envs/public/harbor-config.yaml

```diff
@@ -33,6 +33,7 @@ data:
         harbor:
           annotations:
             cert-manager.io/issuer: letsencrypt
+            nginx.ingress.kubernetes.io/limit-rps: "25"
```

Comment:
Nginx doc says that NGINX returns status code 503 Service Unavailable when the limit is reached. 
It is the case of the k8s-ingress-controller too? If yes, consider setting 429 instead.

Reply:
This status code can be changed via ingress nginx config map, I added an example in the docs. See eb378bf06c29cf4e91f5e123d12592598d5bd278